src/main.rs

```diff
@@ -17,11 +17,22 @@
 #![allow(clippy::too_many_lines)]
 
 #[cfg(any(
+    all(
+        feature = "native-roots",
+        feature = "webpki-roots",
+        feature = "native-tls"
+    ),
```

Comment:
This should already be covered by the cases below, right?

Reply:
True!